### DISCUSSION TITLE
Textboxes don't play a sound if you clear them with the Esc key

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#20099] [Q&A] [answered]
It seems to be the last case without a sound yet.
Maybe the erase/delete/backspace sound is enough (that's what it does, ideally), but not hearing a sound also makes sense since you have "undone" what you were writing…

https://user-images.githubusercontent.com/43073074/188123904-28d5f75a-b726-4387-b12d-91099c3f7c85.mp4

**Comment by Theighlin** [comment, marked as answer]
Would have probably been better to post as comment here https://github.com/ppy/osu/issues/14082 (as you yourself have mentioned in there it's better than opening new issues for any missing sound).

**Reply by Gabixel** [reply]
Fair point. For some reason, I forgot about that post.
